firefox_desktop/views/crash_usage.view.lkml

```diff
@@ -0,0 +1,369 @@
+view: crash_usage {
+  sql_table_name: mozdata.telemetry.clients_daily ;;
```

Comment:
I haven't seen the pattern of multiple views pointing to the same table much but I think it's something to think about. No strong feelings about it though

Reply:
Good point. IMO it simplifies use of this explore, and let's us simplify it for the end users without adding a bunch of complicated fields they will never need.